internal/peer/node/start.go

```diff
@@ -1076,6 +1072,18 @@ func computeChaincodeEndpoint(chaincodeAddress string, chaincodeListenAddress st
 	return ccEndpoint, nil
 }
 
+func createDockerClient(coreConfig *peer.Config) (client *docker.Client, err error) {
+	if coreConfig.VMDockerTLSEnabled {
+		client, err = docker.NewTLSClient(coreConfig.VMEndpoint, coreConfig.DockerCert, coreConfig.DockerKey, coreConfig.DockerCA)
+	} else {
+		client, err = docker.NewClient(coreConfig.VMEndpoint)
+	}
+	if err != nil {
+		logger.Panicf("cannot create docker client: %s", err)
+	}
```

Comment:
I think this should be moved back up in the block at line 500

Reply:
Oops, agreed.

Reply:
Done